src/CommunityToolkit.Maui.Core/Views/DrawingView/Service/DrawingViewService.windows.cs

```diff
@@ -173,4 +170,46 @@ static void DrawInk(this CanvasDrawingSession session, IEnumerable<InkStroke> st
 			}
 		}
 	}
+
+	static ICanvasBrush GetCanvasBrush(ICanvasResourceCreator canvasResourceCreator, Paint? brush, Windows.Foundation.Size size)
+	{
+		switch (brush)
+		{
+			case SolidPaint solidColorBrush:
+				return new CanvasSolidColorBrush(canvasResourceCreator, solidColorBrush.Color.ToWindowsColor());
+			case LinearGradientPaint linearGradientBrush:
+				{
+					var gradientStops = new CanvasGradientStop[linearGradientBrush.GradientStops.Length];
+					for (var index = 0; index < linearGradientBrush.GradientStops.Length; index++)
+					{
+						var gradientStop = linearGradientBrush.GradientStops[index];
+						gradientStops[index] = new CanvasGradientStop(gradientStop.Offset, gradientStop.Color.ToWindowsColor());
+					}
+
+					return new CanvasLinearGradientBrush(canvasResourceCreator, gradientStops)
+					{
+						StartPoint = new Vector2((float)(linearGradientBrush.StartPoint.X * size.Width), (float)(linearGradientBrush.StartPoint.Y * size.Height)),
```

Comment:
why in some places do you use the property `Size.Height` and in others do you use the field `Size._height`? Would be nice to have consistence

Reply:
To avoid casting. Height is double, _height is float.